site/profile/manifests/cvmfs.pp

```diff
@@ -22,19 +25,26 @@
     require => [Package['cvmfs-repo'], Package['cc-cvmfs-repo']]
   }
 
+  if 'ref.mugqic' in $repositories {
+    file { '/etc/cvmfs/config.d/ref.mugqic.conf':
+      ensure  => 'present',
+      content => epp('profile/cvmfs/ref.mugqic.conf')
+    }
+  }
+
   file { '/etc/cvmfs/default.local.ctmpl':
     ensure  => 'present',
     content => epp('profile/cvmfs/default.local', {
       'quota_limit'  => $quota_limit,
-      'repositories' => $repositories,
+      'repositories' => $repositories_no_alien_cache ,
```

Comment:
If this line sets CVMFS_REPOSITORIES , it should include all repositories, whether or not they use an alien cache.